CONTRIBUTING.md

```diff
@@ -13,6 +13,49 @@ and let our peers review the code before it gets merged.
 - We try to prefix our commit messages with a word that quickly tells the reader where the change happened. Examples are `editor`, `doc`, `<component-name>`, etc.
 - Changes that touch the core project should be unit-tested. 
 
+### Unity License Setup For Automated Testing
+
+All Pull Request must pass an automated run of all the unit tests before merging. These will fail until your fork of the `VisualPinball.Engine` is configured with Unity's license information.

```

Comment:
I would formulate is less restrictive. We'll still merge PRs when it's just doc or something that we test locally. Maybe:

```suggestion
It's preferred to make the automated tests run when creating your PR. Since Unity needs a license key (which can be obtained with a free account), you'll need to configure your fork to use the correct secrets.
```